### ISSUE TITLE
"Home, Upload, About" - either have it or not. Also, if we keep it, it should be center-aligned

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#197]
## User story
1. As a user
2. I want actually have content in each page or not have the page at all
3. So that ...

## Acceptance criteria
* Brief information about project/product
* User guide information

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
